src/platforms/zulip/triagebot.md

```diff
@@ -23,7 +23,8 @@ You can drop your claim to the issue via `@rustbot release-assignment`; Rust tea
 
 `@rustbot assign @user` can be used only by Rust team members and will assign that user to the issue (with same rules as before -- either directly or indirectly).
 
-Soon (when the "highfive" bot migration will be complete, see [rust-lang/highfive#258](https://github.com/rust-lang/highfive/pull/258)), `r?` will also assign reviewers to PRs, though unlike issues, non-team members cannot be assigned. Anyone can invoke the command.
+The assignment handler also handles automatic assignment of PR reviewers and for the `r?` command to reassign reviewers.
+See [the Assignment documentation](https://github.com/rust-lang/triagebot/wiki/Assignment) for more details.
```

Comment:
I would prefer if there was only one set of documentation for triagebot. Would anyone object to unifying it? I see a few options:
1. Delete the wiki, move everything here.
2. Delete this page, and move everything to the wiki.
3. Delete the wiki and this page, and move everything to an mdbook in the triagebot repo, hosted on GitHub Pages.

I would prefer option 3, or 1. Let me know if you would be interested in any of these.

Reply:
I personally much prefer using forge over GitHub wikis as it's easier to collaborate on and to unify across the entire project. I'd love for forge's usage to grow so while I see the appeal of having an mdbook in the triagebot repo, I think it's better to keep things in forge. That being said, my only strong reaction is to not use a wiki.